ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 import NotificationCenter
 import WMF
 
-class WMFTodayContinueReadingWidgetViewController: UIViewController, NCWidgetProviding {
+class WMFTodayContinueReadingWidgetViewController: UIViewController, NCWidgetProviding, Themeable {
```

Comment:
Seems like you could make this subclass ExtensionViewController instead.

Reply:
agreed - missed this class when I refactored